server/owlEye/app/index.py

```diff
@@ -60,35 +53,54 @@ def handleRequestFromAPIGateway(event):
 
 def handleRequestFromSQS(event):
     try:
+        body = json.loads(event["Records"][0]["body"])
+        fileIdx = int(body["fileIdx"])
+        jobID = body["jobID"]
+        key = body["fileKey"]
+        # 0. validate 
+        validateFileStatus(jobID, fileIdx, key)
         # 0. init clients 
         s3 = boto3.client('s3')
         bucket = os.getenv("SRC_BUCKET", "visual-testing-backend-v2-srcbucket-p3rsmcrs75qa")
```

Comment:
```suggestion
        try:
            bucket = os.environ["SRC_BUCKET"]
        except KeyError:
            bucket = "visual-testing-backend-v2-srcbucket-p3rsmcrs75qa"
```
Same dealio here. Use a try except clause to handle KeyError and set a default value